galsim/base.py

```diff
@@ -338,6 +338,258 @@ def getGSParams(self):
         """
         return self.SBProfile.getGSParams()
 
+    def calculateHLR(self, size=None, scale=None, centroid=None, flux_frac=0.5):
```

Comment:
Mention these new features in CHANGELOG? (nothing to do with the specifics of this line of code, sorry)